api/sixtyfps-rs/sixtyfps-build/lib.rs

```diff
@@ -61,16 +61,17 @@ pub struct CompilerConfiguration {
     config: sixtyfps_compilerlib::CompilerConfiguration,
 }
 
-impl CompilerConfiguration {
-    /// Creates a new default configuration.
-    pub fn new() -> Self {
+impl Default for CompilerConfiguration {
```

Comment:
Hmm, this changes public API in a source incompatible way. I suggest to move that out of this merge request and we discuss it separately.

Reply:
I dropped this one now.